backend/src/main/java/codesquad/airbnb/reservation/StartEndDate.java

```diff
@@ -1,18 +1,24 @@
 package codesquad.airbnb.reservation;
 
-import lombok.AllArgsConstructor;
+import codesquad.airbnb.accomodation.exception.InvalidStartEndDateException;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
 
 import javax.persistence.Embeddable;
-import java.time.LocalDateTime;
+import java.time.LocalDate;
 
 @Embeddable
 @Getter
-@AllArgsConstructor
 @NoArgsConstructor // for @Embeddable
 public class StartEndDate {
 
-    private LocalDateTime startDate;
-    private LocalDateTime endDate;
+    private LocalDate startDate;
+    private LocalDate endDate;
+
+    public StartEndDate(LocalDate startDate, LocalDate endDate) {
+        this.startDate = startDate;
+        this.endDate = endDate;
+
+        if ((startDate != null && endDate != null) && startDate.isAfter(endDate)) throw new InvalidStartEndDateException();
+    }
```

Comment:
검증을 생성자 상단에서 하면 더 좋을듯 합니다 :)
```suggestion
    public StartEndDate(LocalDate startDate, LocalDate endDate) {
        if ((startDate != null && endDate != null) && startDate.isAfter(endDate)) throw new InvalidStartEndDateException();
        this.startDate = startDate;
        this.endDate = endDate;
    }
```

Reply:
검증 메소드를 분리해주면 더 좋을거 같구요
```suggestion
    public StartEndDate(LocalDate startDate, LocalDate endDate) {
        validate(startDate, endDate);
        this.startDate = startDate;
        this.endDate = endDate;
    }
```